.github/workflows/lint.yaml

```diff
@@ -33,3 +33,13 @@ jobs:
       run: |
         curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.55.2
         make lint
+
+    - name: Install Vacuum
+      # See: https://quobix.com/vacuum/installing/
+      run: |
+        curl -fsSL https://quobix.com/scripts/install_vacuum.sh | sh > /dev/null
```

Comment:
piping shell from curl seems to be wrong. This seems also a personal website from which we would fetch the script.
I wonder why we can't just do `go run https://github.com/daveshanley/vacuum` or `go install github.com/daveshanley/vacuum@latest`.

Right now it would be a blocking issue given the time after seeing too many supply-chain attacks in the wild.

Reply:
+1 on avoiding this.